LOC.json

```diff
@@ -56985,38 +56985,51 @@
     "airportICAO": "KCAK",
     "identifier": "ICAK",
     "associatedRunwayNumber": "01",
-    "latitude": 40.93267200,
-    "longitude": -81.43990300,
-    "elevation": 1204,
+    "latitude": 40.90718757,
+    "longitude": -81.44135808,
+    "elevation":  374.3,
```

Comment:
Elevation should be indicated in **feet**! Also, remove that space you created; it might have been the reason Travis failed...